controllers/articles-controller.js

```diff
@@ -9,13 +10,29 @@ exports.getArticles = (req, res, next) => {
     .then((article) => {
       res.status(200).send({ article });
     })
-    .catch(next);
+    .catch((err) => {
+      next(err);
+    });
 };
 
 exports.getAllArticles = (req, res, next) => {
   selectAllArticles()
     .then((articles) => {
       res.status(200).send({ articles });
     })
-    .catch(next);
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.patchArticleVotes = (req, res, next) => {
+  const articleId = Number(req.params.article_id);
```

Comment:
Why do you need to convert the id to a number here? It will become a string once you put it in the query anyway.

Reply:
<img width="467" alt="Screenshot 2024-08-27 at 14 17 06" src="https://github.com/user-attachments/assets/9c174a64-424a-42a4-b8ed-7dbb68323261">